src/fields/time.coffee

```diff
@@ -11,9 +11,9 @@ FormRenderer.Models.ResponseFieldTime = FormRenderer.Models.ResponseField.extend
     "#{@getValue().hours || '00'}:#{@getValue().minutes || '00'}:#{@getValue().seconds || '00'} #{@getValue().am_pm}"
 
   validateType: ->
-    hours = parseInt(@get('value.hours'), 10) || 0
-    minutes = parseInt(@get('value.minutes'), 10) || 0
-    seconds = parseInt(@get('value.seconds'), 10) || 0
+    hours = parseInt(@get('value.hours'), 10)
+    minutes = parseInt(@get('value.minutes'), 10)
+    seconds = parseInt(@get('value.seconds'), 10)
```

Comment:
I thought the decision was to default seconds to 0?

Reply:
Oh yep, that would explain why the time-related test started failing. Not sure how this change snuck in when I branched off the time PR for this one. Pushed up a fix to revert this.